soccer/gameplay/plays/restarts/placement.py

```diff
@@ -59,8 +59,8 @@ def on_enter_placing(self):
                                  required=True,
                                  priority=90)
 
-        #line_up = tactics.line_up.LineUp(self.create_lineup())
-        #self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
+        line_up = tactics.line_up.LineUp(self.create_lineup())
+        self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
```

Comment:
We need to think carefully about this. By the rules, we need to stay 'out of the way' of the ball placement, so this is probably safer. I would assume we would have a bit of free 'stopped' time to get ready, so I don't think this would be a problem. This is definitely something that needs to be looked into closer though, thanks for bringing it up! 👍